test/unit/visual/visualTest.js

```diff
@@ -151,6 +151,12 @@ window.visualTest = function(
         actual.push(myp5.get());
       });
 
+
+      if (actual.length === 0) {
+        throw new Error('No screenshots were generated. Check if your test generates screenshots correctly.If the test includes asynchronous operations, ensure they complete before the test ends.');
```

Comment:
Looks like a space is missing here:
```suggestion
        throw new Error('No screenshots were generated. Check if your test generates screenshots correctly. If the test includes asynchronous operations, ensure they complete before the test ends.');
```